.travis.yml

```diff
@@ -1,21 +1,20 @@
 language: go
 sudo: required
 go:
-- 1.5
+- 1.6.2
 services:
   - docker
 before_install:
 - rm -rf ~/.nvm && git clone https://github.com/creationix/nvm.git ~/.nvm && (cd ~/.nvm && git checkout `git describe --abbrev=0 --tags`) && source ~/.nvm/nvm.sh && nvm install $TRAVIS_NODE_VERSION
 - npm install
 - npm run build
 - pip install --user ruamel.yaml
-- wget https://s3.amazonaws.com/travis-phantomjs/phantomjs-2.0.0-ubuntu-12.04.tar.bz2
-- tar -xjf phantomjs-2.0.0-ubuntu-12.04.tar.bz2
-- export PHANTOMJS_BIN=./phantomjs
+- go get github.com/kardianos/govendor
```

Comment:
and i forgot that, if you use godep with go1.6 and greater, you have to do weird stuff that's anti go1.6 like not use the vendoring component which will make it weirder to use on a developer's machine http://docs.cloudfoundry.org/buildpacks/go/index.html